.github/workflows/build.yml

```diff
@@ -95,3 +94,62 @@ jobs:
         SLACK_TITLE: Build Failure
         SLACK_MESSAGE: ':alert: Build failure on ${{ github.event.inputs.environment }} :sadparrot:'
         SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK }}
+
+  axe-tests:
```

Comment:
Can this go in its own file with a name?

Reply:
It can but this way it uses the cached version of the docker, as well as only commercing if its viable, which in turns add extend the duration. I'm not sure of the available git action time allocate hence opted for this middle ground.

We also need to revisit this `build` job and it's usage of `docker` and `docker-compose` as a sum, and break this up.